addons/website/static/src/builder/plugins/options/google_maps_option/google_maps_option_plugin.js

```diff
@@ -68,6 +68,17 @@ export class GoogleMapsOptionPlugin extends Plugin {
 
         /** @type {Map<HTMLElement, Deferred} */
         this.recentlyDroppedSnippetDeferredInit = new Map();
+        this.upgradeSnippets();
+    }
+
+    // TODO: Remove this method when data-vxml is reintroduced.
+    upgradeSnippets() {
+        // This is for pages which already existed before the plugin was created.
+        this.document
+            .querySelectorAll(".s_google_map")
+            .forEach((mapSnippetEl) => {
+                mapSnippetEl.classList.add("o_not_editable");
```

Comment:
Also set the new `dataset.vxml` otherwise they will be considered as obsolete while they have been adapted.